justfile

```diff
@@ -1,11 +1,14 @@
 # To use this file, install Just: cargo install just
 set dotenv-load
+# eports all justfile variables as environment variables
```

Comment:
nit: typo `exports`

Reply:
I'm not sure whether I'm in favour of these, variables such as line length are exported with this, and there's no point for that?
IMHO it's better to stick to the previously established convention with the substitutions.